deploy.tf

```diff
@@ -0,0 +1,248 @@
+      ]
+    }
+  }
+}
+
+resource "azurerm_network_profile" "networkprofile" {
+  name                = "${var.name}networkprofile"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+
+  container_network_interface {
+    name = "containernic"
+
+    ip_configuration {
+      name      = "ipconfig"
+      subnet_id = azurerm_subnet.backend.id
+    }
+  }
+}
+
+resource "azurerm_container_group" "cg" {
+  name                = "${var.name}cg1234"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  ip_address_type     = "private"
+  network_profile_id  = azurerm_network_profile.networkprofile.id
+  os_type             = "Linux"
+
+  image_registry_credential {
+      server    = azurerm_container_registry.acr.login_server
+      username  = azurerm_container_registry.acr.admin_username
+      password  = azurerm_container_registry.acr.admin_password
+  }
+
+  diagnostics {
+      log_analytics {
+          workspace_id  = azurerm_log_analytics_workspace.la.workspace_id
+          workspace_key = azurerm_log_analytics_workspace.la.primary_shared_key
+      }
+  }
+
+  container {
+    name   = "orleanshost"
+    image  = "${var.name}registry.azurecr.io/backend:v1"
+    cpu    = "1.0"
+    memory = "1.5"
+
+    environment_variables = {
+        "ORLEANS_CONFIG"          = "STORAGE"
+        "StorageConnectionString" = azurerm_storage_account.storage.primary_connection_string
+    }
+
+    ports {
+      port     = 11111
+      protocol = "TCP"
+    }
+
+    ports {
+      port     = 30000
+      protocol = "TCP"
+    }
+  }
+
+  depends_on = [
+    null_resource.acrimagebuildpush
+  ]
+}
+
+resource "azurerm_monitor_diagnostic_setting" "metricsoutput" {
+  name                          = "${azurerm_container_group.cg.name}metricsoutput"
+  target_resource_id            = azurerm_container_group.cg.id
+  log_analytics_workspace_id    = azurerm_log_analytics_workspace.la.id
+
+  metric {
+    category = "AllMetrics"
+    retention_policy {
+      enabled   = true
+      days      = 7
+    }
+  }
+}
+
+resource "azurerm_app_service_plan" "appserviceplan" {
+  name                = "${var.name}appserviceplan"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Standard"
+    size = "S1"
+  }
+}
+
+resource "azurerm_app_service" "appservice" {
+  name                = "${var.name}appservice"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  app_service_plan_id = azurerm_app_service_plan.appserviceplan.id
+
+  identity {
+    type  = "SystemAssigned"
+  }
+
+  site_config {
+    app_command_line  = ""
+    always_on         = true
+    linux_fx_version  = "DOCKER|${var.name}registry.azurecr.io/frontend:v1"
```

Comment:
I suspected this is what this field was for. Glad to see that provides a means to set containers. Out of scope for this PR but, would be curious what happens when we need to push a new version?

Reply:
I think we either solve that by using a "latest" tag in ACR or we could simply re-deploy the TF... I expect it would update the app service which would restart it and pull the new image.